theme.json

```diff
@@ -1192,26 +1192,53 @@
 			"core/quote": {
 				"border": {
 					"style": "solid",
-					"width": "0 0 0 2px",
+					"width": "0 2px 0 2px",
 					"color": "var:preset|color|contrast"
 				},
 				"spacing": {
+					"blockGap": "var:preset|spacing|30",
+					"margin": {
+						"left": "0",
+						"right": "0"
+					},
 					"padding": {
+						"top": "var:preset|spacing|30",
+						"right": "var:preset|spacing|40",
 						"bottom": "var:preset|spacing|30",
-						"left": "var:preset|spacing|40",
-						"top": "var:preset|spacing|30"
+						"left": "var:preset|spacing|40"
 					}
 				},
 				"typography": {
-					"fontStyle": "normal",
+					"fontSize": "var:preset|font-size|medium",
+					"lineHeight": "1.1",
 					"fontWeight": "600"
 				},
 				"elements": {
 					"cite": {
 						"typography": {
-							"fontSize": "var:preset|font-size|small",
+							"fontSize": "0.65em",
```

Comment:
Why was this size changed again, why is  it using `em` and not `rem`?
It is an actual question, not a complaint or request 😄

Reply:
One would think the "task force" that has updated a large number of quote blocks in the old bundled themes the last 6 months would know why 😆 
It's so that the cite changes size relatively to whatever size the user sets on the quote.